features/old/accounts/personal_details.feature

```diff
@@ -41,10 +41,10 @@ Feature: Personal Details
     And I fill in "Username" with ""
     And I fill in "Current password" with "supersecret"
     And I press "Update Details"
-    Then I should see inline error "is too short (minimum is 3 characters)" for user username input
+    Then "Username" shows error "is too short (minimum is 3 characters)"
```

Comment:
This wording is confusing to me

Reply:
Yeah, sounds weird. I would say `"Username" field has inline error "xxx"`

Reply:
👍🏻

Reply:
OK but this step was approved and merged months ago: https://github.com/3scale/porta/pull/3613/files#diff-78e6932955a83b3381c51991e615ce886841d8a971d0f3b2052d0eeee282d6b7R23-R25

Reply:
Renamed